src/client.ts

```diff
@@ -167,6 +167,25 @@ export default class Client extends EventEmitter {
     this.dispatch.close();
   }
 
+  isStreamIceConnected = (local: boolean) => {
+    let connected = true;
+    const failedStatuses = ["closed", "disconnected", "failed"]
+    const streams = local ? this.localStreams as Stream[] : Object.values(this.streams) as Stream[];
```

Comment:
`Object.values` is a newer Javascript method and so the compiled output of ion-sdk-js won't work on older browsers like IE. This is why we have our own implementation in our codebase. I would avoid using it for compatibility reasons.

Reply:
Interesting, we do already have Object.values() in code. Its even in official repo https://github.com/pion/ion-sdk-js/blob/master/src/client.ts#L130

Reply:
In addition, IE doesn't support WebRTC anyway :-)